app/(storyblok)/[[...slug]]/page.tsx

```diff
@@ -22,12 +22,33 @@ type ParamsType = {
 // Control what happens when a dynamic segment is visited that was not generated with generateStaticParams.
 export const dynamicParams = false;
 
+// Storyblok bridge options.
+const bridgeOptions = {
+  preventClicks: true,
+  resolveLinks: 'story',
+};
+
+/**
+ * Init on the server.
+ */
+storyblokInit({
+  accessToken: process.env.STORYBLOK_ACCESS_TOKEN, // Preview token because this is in server side.
+  use: [apiPlugin],
+  apiOptions: {
+    region: 'us',
+  },
+  components: Components,
+  enableFallbackComponent: true,
+  customFallbackComponent: (component) => {
+    return <ComponentNotFound component={component} />;
+  },
+});
```

Comment:
This initializes the connection with Storyblok on the server side. Here, we can use the preview key. 
I've also included the fallback component here to match the front-end initialization so that there isn't a mismatch in the response from the server and hydration issues.

Reply:
This is really nice - thank you! 🚀